test/suites/lib/skip.py

```diff
@@ -0,0 +1,24 @@
+import functools
+import pkg_resources
+import re
+
+SQL_SUPPORT_TNT_VERSION = '2.0.0'
+
+
+def skip_or_run_sql_test(func):
+    @functools.wraps(func)
+    def wrapper(self, *args, **kwargs):
+        tnt_version_str = re.match(
+            r'[\d.]+', self.srv.admin('box.info.version')[0]
+        ).group()
```

Comment:
It works for 2.0.0 border, but if we'll need, say, 2.10.0-beta1 border, we'll in a trouble. Not important here, but worth to leave a comment.

Reply:
Ok, thanks.